engines/python/setup/djl_python/tensorrt_llm.py

```diff
@@ -50,7 +50,7 @@ def inference(self, inputs: Input) -> Output:
         """
         outputs = Output()
 
-        input_data, input_size, parameters, errors, batch = parse_input(
+        input_data, input_size, parameters, errors, batch, _ = parse_input(
```

Comment:
This change break backward compatibility

Reply:
Do you mean for people using parse_input in their custom handlers?

Reply:
As discussed offline, will introduce another function, not to break the backward compatibility of the users.